### PR TITLE
feat: Add Acknowledgements section in Device Settings

### LINKED ISSUE #1220: Add Acknowledgements in Device Settings
### Feature Request

**Feature Description**
There can be an acknowledgements section in similar to EventBrite where all the open source libraries used in the app can be listed.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1220 
Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes:  Added an acknowledgements section in Device Settings to view all the Open Source libraries used in the project. This section has been made similar to EventBrite.

Screenshots for the change:
ezgif com-resize 35

### CODE REVIEW

**Comment by sridharjajoo** [comment]
@iamareebjamal @Masquerade0097 @mishuvs Please review

File: app/src/main/java/org/fossasia/openevent/app/core/settings/SettingsFragment.java [hunk 1]
```diff
@@ -52,6 +53,11 @@ public void onCreatePreferencesFix(@Nullable Bundle bundle, String rootKey) {
             }
             return true;
         });
+
+        findPreference("acknowledgement").setOnPreferenceClickListener(preference -> {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/settings/SettingsFragment.java]: getString

**Comment by Masquerade0097** [comment]
@sridharjajoo EventBrite shows the url for the licences as well in a new page, when clicked. How about adding that or dmaybe irectly opening them in browser ?

**Comment by sridharjajoo** [comment]
@Masquerade0097 Actually it is taking place here as well. I haven't shown it in the GIF.

**Comment by sridharjajoo** [comment]
@Masquerade0097 Please merge this approved P.R

**Pull request merged**